src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -155,7 +154,6 @@ class BaseCreateCourseForm extends React.Component {
             timeLabel={`Start time (${localTimeZone})`}
             helpText={startDateHelp}
```

Comment:
The help text should probably be updated to specify that the day and month need to be two digits and the year needs to be 4 so users don't run into the situation where "1-1-2020" isn't saveable.